ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -821,6 +830,7 @@ let disable_machine_account ~service_name = function
       in
       try
         Helpers.call_script ~env net_cmd args |> ignore ;
+        config_winbind_damon None ;
```

Comment:
Using `config_winbind_damon ~workgroup:None ~netbios_name:None ~domain:None ;`

Would make the intent of the call clearer.